### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JGVR**
updated the data design section of the ReadMe. Also, modify the UML diagrams for both the collection and data classes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
+PortfolioAPI is a RESTful service designed to seamlessly integrate with MongoDB, enabling efficient CRUD operations across various collections. It serves as the backbone for user interfaces that display comprehensive portfolio information, including work experiences and projects, tailored for individuals looking to showcase their professional journey. Unique to our API is the integration of a sophisticated Chatbot, powered by OpenAI's latest GPT-4o model, which acts as a personal assistant, enriching user interaction with instant, intelligent responses to inquiries. Developed with **django, pymongo, django channels, djangorestframework, pydantic, langchain**, this API offers an unparalleled blend of functionality and user engagement.
```
Comment (hecvasro) [inline README.md]:
```suggestion
PortfolioAPI is a RESTful service designed to seamlessly integrate with MongoDB, enabling efficient CRUD operations across various collections. It serves as the backbone for user interfaces that display comprehensive portfolio information, including work experiences and projects, tailored for individuals looking to showcase their professional journey. Unique to our API is the integration of a sophisticated Chatbot, powered by OpenAI's latest GPT-4o model, which acts as a personal assistant, enriching user interaction with instant, intelligent responses to inquiries.
```

I don't think you have to provide these lower-level details in the project's summary.

**Review by hecvasro — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (JGVR) [reply README.md]: I made the changes we discussed during our call today. Please review and approve if all looks good @hecvasro

File: README.md [hunk 2]
```diff
@@ -1,21 +1,326 @@
...
+* ATLAS_CONN_STR
+* ATLAS_DB_NAME
+* ATLAS_VECTOR_COLLLECTION
+* ATLAS_VECTOR_INDEX_NAME
+* ATLAS_EMBEDDING_KEY
+* OPENAI_API_KEY
+* OPENAI_EMBEDDING_MODEL
+* OPENAI_CHAT_MODEL
+* LANGCHAIN_PROJECT
+* LANGCHAIN_API_KEY
```
Comment (hecvasro) [inline README.md]: To which file should I add these env vars?

**Review by hecvasro — APPROVED** [review]
@JGVR, looks good to me!

**Pull request merged**